src/components/Toolbar.js

```diff
@@ -25,15 +25,39 @@ const clearHoldTimer = () => {
 (() => {
   window.addEventListener('mouseup', clearHoldTimer)
   window.addEventListener('touchend', clearHoldTimer)
+  window.addEventListener('resize', () => {
+    const toolbarElement = document.getElementById('toolbar')
+    const leftArrowElement = document.getElementById('leftArrow')
+    const window90 = window.innerWidth * 0.9
+    if (toolbarElement.scrollWidth > window90) leftArrowElement.className = ''
+    else leftArrowElement.className = 'hidden'
+  })
+  window.onload = () => {
+    const toolbarElement = document.getElementById('toolbar')
+    const leftArrowElement = document.getElementById('leftArrow')
+    const rightArrowElement = document.getElementById('rightArrow')
+    const initialScrollLeft = toolbarElement.scrollLeft
+    const window90 = window.innerWidth * 0.9
+
+    if (toolbarElement.scrollWidth > window90) leftArrowElement.className = ''
+
+    toolbarElement.onscroll = () => {
+      // console.log(toolbarElement.scroll)
+      if (toolbarElement.scrollLeft === 0) leftArrowElement.className = 'hidden'
+      else leftArrowElement.className = ''
+      if (toolbarElement.scrollLeft < initialScrollLeft) rightArrowElement.className = ''
+      else if (toolbarElement.scrollLeft >= initialScrollLeft) rightArrowElement.className = 'hidden'
+    }
```

Comment:
Is there a way to do this with conventional React? It seems ugly to have to resort to direct DOM manipulations.

Reply:
Ok, I suppose I got carried. I'll make sure to remember this is a react project.